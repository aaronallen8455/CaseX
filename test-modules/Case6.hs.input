module Case6 where

main :: IO ()
main = pure ()

test1 :: Bool
test1 =
 case 1 of
  SPLIT -> False

test2 :: Bool
test2 =
 case 1.1 of
  SPLIT -> False

test3 :: Bool
test3 =
 case 'a' of
  SPLIT -> False
