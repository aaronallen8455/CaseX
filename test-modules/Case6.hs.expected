module Case4 where

main :: IO ()
main = pure ()

test1 :: Bool
test1 =
 case 1 of
  1 -> False

test2 :: Bool
test2 =
 case 1.1 of
  1.1 -> False
